src/modules/dashboard/components/TransferFundsDialog/TransferFundsDialogForm.tsx

```diff
@@ -226,67 +237,66 @@ const TokensMoveDialogForm = ({
         <Heading
           appearance={{ size: 'medium', margin: 'none' }}
           text={MSG.title}
+          className={styles.title}
         />
       </DialogSection>
       <DialogSection>
-        <Select options={domainOptions} label={MSG.from} name="fromDomain" />
-        {!!tokenAddress && (
-          <div className={styles.domainPotBalance}>
-            <FormattedMessage
-              {...MSG.domainTokenAmount}
-              values={{
-                amount: (
-                  <Numeral
-                    appearance={{
-                      size: 'small',
-                      theme: 'grey',
-                    }}
-                    value={fromDomainTokenBalance || 0}
-                    unit={getTokenDecimalsWithFallback(
-                      selectedToken && selectedToken.decimals,
-                    )}
-                    truncate={3}
-                  />
-                ),
-                symbol: (selectedToken && selectedToken.symbol) || '???',
-              }}
-            />
-          </div>
-        )}
-      </DialogSection>
-      <DialogSection>
-        <Select options={domainOptions} label={MSG.to} name="toDomain" />
-        {values.toDomain !== undefined && !!values.tokenAddress && (
-          <div className={styles.domainPotBalance}>
-            <FormattedMessage
-              {...MSG.domainTokenAmount}
-              values={{
-                amount: (
-                  <Numeral
-                    appearance={{
-                      size: 'small',
-                      theme: 'grey',
-                    }}
-                    value={toDomainTokenBalance || 0}
-                    unit={getTokenDecimalsWithFallback(
-                      selectedToken && selectedToken.decimals,
-                    )}
-                    truncate={3}
-                  />
-                ),
-                symbol: (selectedToken && selectedToken.symbol) || '???',
-              }}
+        <div className={styles.domainSelects}>
+          <div>
+            <Select
+              options={domainOptions}
+              label={MSG.from}
+              name="fromDomain"
+              appearance={{ theme: 'grey' }}
             />
+            {!!tokenAddress && (
+              <div className={styles.domainPotBalance}>
+                <FormattedMessage
+                  {...MSG.domainTokenAmount}
+                  values={{
+                    amount: (
+                      <Numeral
+                        appearance={{
+                          size: 'small',
+                          theme: 'grey',
+                        }}
+                        value={fromDomainTokenBalance || 0}
```

Comment:
The default value could have been put inside the `useMemo` hook, but I guess this is fine as well